server/service/src/preference/preferences/use_simplified_mobile_ui.rs

```diff
@@ -0,0 +1,23 @@
+use crate::preference::{PrefKey, Preference, PreferenceType, PreferenceValueType};
+
+pub struct UseSimplifiedMobileUi;
+
+impl Preference for UseSimplifiedMobileUi {
+    type Value = bool;
+
+    fn key(&self) -> PrefKey {
+        PrefKey::UseSimplifiedMobileUi
+    }
+
+    fn preference_type(&self) -> PreferenceType {
+        PreferenceType::Store
+    }
+
+    fn value_type(&self) -> PreferenceValueType {
+        PreferenceValueType::Boolean
+    }
+
+    fn default_value(&self) -> Self::Value {
+        true
+    }
```

Comment:
What do we think about having a default value of `true` for this preference? To me it makes sense, as it still won't have any effect unless the screen size is appropriate.

Reply:
I like it...